mcgill-fr.csl

```diff
@@ -19,1111 +17,716 @@
+            <!-- COMMENTED OUT FOR 9TH ED  <text variable="authority" prefix=" (" suffix=")" strip-periods="true"/> -->
           </else>
         </choose>
-        <text macro="issued" prefix="(" suffix=")"/>
-        <text variable="page"/>
-      </group>
-    </group>
-    <text macro="URL"/>
+
+    <!-- COMMENTED OUT FOR 9TH ED  <text variable="URL" prefix=" (available on " suffix=")"/> -->
   </macro>
-  <macro name="note-article-journal">
+  <macro name="pinpoint">
     <group delimiter=" ">
-      <text macro="contributors-note" suffix="," strip-periods="true"/>
-      <text variable="title" quotes="true"/>
       <choose>
-        <if variable="volume" match="none">
-          <text macro="issued-year" prefix="[" suffix="]"/>
-          <number variable="issue"/>
+        <if locator="page">
+          <choose>
+            <if variable="locator">
+              <label variable="locator" plural="contextual" form="short" strip-periods="true" prefix=" "/>
+		      <text variable="locator"/>
+			 
+            </if>
+          </choose>
         </if>
+		<else-if locator="section">
+		   <label variable="locator" plural="contextual" form="short" strip-periods="true" prefix=", "/>
+		   <text variable="locator"/>
+		</else-if>
+		<else-if locator="sub-verbo">
+		   <label variable="locator" plural="contextual" form="short" strip-periods="true" font-style="italic" prefix=" "/>
+		   <text variable="locator" quotes="true" />
+		</else-if>
         <else>
-          <text macro="issued-year" prefix="(" suffix=")"/>
-          <group delimiter=":">
-            <text variable="volume"/>
-            <number variable="issue"/>
-          </group>
+		  <label variable="locator" plural="contextual" form="short" strip-periods="true" prefix=" "/>
+		  <text variable="locator"/>
         </else>
       </choose>
-      <text variable="container-title" form="short" strip-periods="true"/>
-      <text variable="page"/>
-    </group>
-    <text macro="point-locators"/>
-    <text macro="URL"/>
-  </macro>
-  <macro name="article-journal">
-    <group delimiter=". ">
-      <text macro="contributors"/>
-      <group delimiter=" ">
-        <text variable="title" quotes="true"/>
-        <choose>
-          <if variable="volume" match="none">
-            <text macro="issued-year" prefix="[" suffix="]"/>
-            <number variable="issue"/>
-          </if>
-          <else>
-            <text macro="issued-year" prefix="(" suffix=")"/>
-            <group delimiter=":">
-              <text variable="volume"/>
-              <number variable="issue"/>
-            </group>
-          </else>
-        </choose>
-        <text variable="container-title" form="short" strip-periods="true"/>
-        <text variable="page"/>
-      </group>
+								
     </group>
-    <text macro="URL"/>
   </macro>
-  <macro name="note-case">
+  <macro name="short-form">
+    <!-- Hump to overcome: cannot check against existence of short title.
```

Comment:
I'm pretty sure `<if variable="title-short">` works, does it not?

Reply:
Hi, and yes - this was an old comment from the previous revision.  It is no longer relevant and I have removed it.

I also did a quick review to remove blank lines and to fix some identation issues.

Thanks for taking the time to review.